packages/site/app/styles/docs-api.css

```
@@ -0,0 +1,116 @@
+/* Fumadocs OpenAPI workbench overrides. Render hooks live in components/docs/openapi-page.tsx. */
+#main-content:has(.site-api-page) > * {
+  width: 100%;
+  max-width: min(100%, 90rem);
+  margin-inline: auto;
+}
+
+.site-api-page {
+  --color-fd-primary: var(--color-fg);
+  --color-fd-primary-foreground: var(--color-canvas);
+  font-size: 0.9375rem;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)